src/calculator.py

```diff
@@ -10,3 +10,24 @@
 #   Multiplication: 8
 #   Division: 2
 #
+
+
+print("Insert first number:")
+n1 = input()
+print("Insert second number:")
+n2 = input()
+
+def func(n1,n2):
+    risultati = []
+    risultati.append(int(n1)+int(n2))
+    risultati.append(int(n1)-int(n2))
+    risultati.append(int(n1)*int(n2))
+    risultati.append(int(n1)/int(n2))
```

Comment:
Approccio interessante. Dato che ripeti il cast a int in ogni append, sarebbe più compatto farlo una sola volta all'inizio della funzione